app/main.py

```diff
@@ -1,6 +1,18 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_storage = {}
+
+    def wrapper(*args, **kwargs) -> Any:
+        key = (args, frozenset(kwargs.items()))
+        if key in cache_storage:
+            print("Getting from cache")
+            return cache_storage[key]
+        else:
```

Comment:
There is no need for `else`, the code will not execute after the `return` statement anyway. Remove redundant `else`